packages/paste-core/components/modal/__tests__/customization.spec.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react';
+
+import {render, screen} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {matchers} from 'jest-emotion';
+
+import {BaseModal, initStyles} from '../stories/customization.stories';
+
+expect.extend(matchers);
+
+jest.mock('@twilio-paste/modal-dialog-primitive', () => {
+  const actual = jest.requireActual('@twilio-paste/modal-dialog-primitive');
```

Comment:
I guess I'm a little unsure on why this needs to be mocked, so providing some sort of commentary on the why might be helpful in a code comment.